cmd/mpi-operator.v1alpha2/app/options/options.go

```diff
@@ -60,6 +61,9 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 
 	fs.BoolVar(&s.PrintVersion, "version", false, "Show version and quit")
 
+	fs.IntVar(&s.MonitoringPort, "monitoring-port", 8443,
```

Comment:
Should this be disabled by default in case prometheus is not available?

Reply:
I think that's all right, prometheus will fetch data from this port. But the default port of monitoring should be 8080?

Reply:
This implementation returns `glog.Error` if the port is not set up for Prometheus: https://github.com/kubeflow/mpi-operator/pull/215/files#diff-58e97bee9d8731245689342bcce48a7dR37

Reply:
@runzhliu See my comment above on `glog.Error`

Reply:
@terrytangyuan I have set the default value 0, so the metrics monitor wouldn't be triggered on.